File: Striver/A-Z/Step 1/while_loop.py

n = int(input())

evenSum = 0
oddSum = 0

while n != 0:
    rem = n % 10
    if rem % 2 == 0:
        evenSum += rem
    else:
        oddSum += rem
    n = n//10

print(evenSum, " ", oddSum)

## Python // performs integer division and / performs float division

File: Striver/A-Z/Step 1/while_loop.cpp
#include<iostream>
using namespace std;

int main() {
	int n;
	cin >> n;

	int oddSum = 0;
	int evenSum = 0;

	while(n != 0){
		int rem = n % 10;
		if(rem % 2 == 0){
			evenSum += rem;
		}else{
			oddSum += rem;
		}
		n = n / 10;
	}
	cout << evenSum << " " << oddSum <<endl;
}
